build.gradle.kts

```diff
@@ -1,29 +1,21 @@
-import java.util.Base64
-import java.util.Properties
+import java.util.*
 
 plugins {
     val kotlinVersion: String by System.getProperties()
     val restriktVersion: String by System.getProperties()
+    val koverVersion: String by System.getProperties()
     java
     `maven-publish`
     signing
-    jacoco
+    id("org.jetbrains.kotlinx.kover") version koverVersion
```

Comment:
Update the Jenkins file to bring back coverage report publishing (or do it on #12)